typing/typedtree.mli

```diff
@@ -215,7 +215,7 @@ and expression_desc =
   | Texp_for of
       Ident.t * Parsetree.pattern * expression * expression * direction_flag *
         expression
-  | Texp_send of expression * meth * expression option
+  | Texp_send of expression * meth
```

Comment:
I've wondered several times what was this `expression option` (and always had to go back to the code to figure out).  Very nice to see it going away.

Reply:
Seconded, I also tried (not very successfully) to figure this out as part of the "recursive value" work.